src/auth/ui/vue/show.ts

```diff
@@ -69,7 +69,6 @@ export class AuthWebview extends VueWebview {
     }
 
     getCredentialFormatError(key: CredentialsKey, value: string | undefined): string | undefined {
-        getLogger().warn('getCredentialFormatError(): %s %s', key, value)
         return getCredentialFormatError(key, value)
```

Comment:
After removing the line, is the error logged in some other fashion (without secret values)?

Reply:
We don't log the error elsewhere in this flow. The actual output of `getCredentialFormatError()` just has an error message but doesn't include the actual secret values.

The object this function returns is displayed in the webview UI to the user.

Reply:
If we aren't logging this failure case at all, we should be... Logs are often the best hope of troubleshooting problems, not screenshots or UI messages